<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1121</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>G.employee</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>G.client</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>G.services</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>G.fournisseurs</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>501</width>
        <height>481</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>481</width>
          <height>291</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajout fournisseur</string>
        </property>
        <widget class="QLineEdit" name="idf">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="nomf">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>120</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="prenomf">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="adressef">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>240</y>
           <width>291</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="emailf">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>50</y>
           <width>191</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="telephonef">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>120</y>
           <width>191</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>50</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID fournisseur</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>180</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom </string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>50</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>E-mail</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>120</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Telephone</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="ajouter_f">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>0</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="musique">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string> ùÑû</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>0</y>
          <width>561</width>
          <height>511</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer les informations d'un fournisseur</string>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID fournisseur</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_sp">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimer_f">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>40</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="exitBtn">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>300</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sendBtn">
         <property name="geometry">
          <rect>
           <x>401</x>
           <y>370</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>339</width>
           <height>380</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <property name="verticalSpacing">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Smtp-server:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Server port:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Recipant to:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Subject:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Message:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="server">
            <property name="text">
             <string>smtp.gmail.com</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="port">
            <property name="text">
             <string>465</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="uname">
            <property name="text">
             <string>mohamedabdelatif.aouadh@esprit.tn</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="paswd">
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>201JMT3405</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="rcpt"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="subject">
            <property name="text">
             <string>Projet By M.Aouadh</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPlainTextEdit" name="msg"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QTableView" name="tabfourn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>511</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="trier">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rech">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>0</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher par id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idrech">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>0</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>190</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>excel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="impc">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>330</y>
        <width>511</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Impdos_2">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer-pdf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>G.produits</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
